PerformanceTesting/ecl/02ba_sortlocal.ecl

```diff
@@ -17,6 +17,6 @@ import suite.perform.files;
 
 ds := files.generateSimple();
 
-s := sort(ds, id3, local, stable(algo));
-
-output(COUNT(NOFOLD(s)) = config.simpleRecordCount);
+s1 := sort(ds, id3, local, stable(algo));
+s2 := SORTED(NOFOLD(s1), id3, local, assert);
```

Comment:
I'm not convinced by this change - in the performance suite - since it will defeat some of the optimizations we can do for executing the output of a sort in parallel.

Tests like these (on smaller numbers of rows) should be in the normal regression suite, and they should have the SORTED check there.